src/witness/witness_calculator.rs

```diff
@@ -285,16 +294,13 @@ impl WitnessCalculator {
     }
 
     pub fn get_witness_buffer(&self) -> Result<Vec<u8>> {
```

Comment:
This method doesn't seem to be used anywhere via a global codebase search, but it is not mentioned by clippy as unused. Maybe clippy is not setup yet.

If we remove this function, I think we could also remove the `store` field from the object

Reply:
Is it actually used? It'd be great to remove the extra field